grr/core/grr_response_core/lib/casing.py

```diff
@@ -0,0 +1,25 @@
+# Lint as: python3
+"""Functions to convert strings between different case styles."""
+import re
+
+
+def SnakeToCamel(snake_str: str) -> str:
+  """Convert a string in snake_case to lowerCamelCase."""
+  if not snake_str:
+    return snake_str
+
+  snake_components = snake_str.split("_")
+  lower_camel_str = (
+    f"{snake_components[0][:1].lower() + snake_components[0][1:]}"
+    f"{''.join([comp.capitalize() for comp in snake_components[1:]])}"
```

Comment:
`map(str.capitalize, snake_components[1:])`

Reply:
Thank you for the functional suggestion! I made the change.